### PR TITLE
Cert tweaks to names and TCB info

### PR DESCRIPTION
**Pull request by nquarton**

### CODE REVIEW

File: rom/dev/src/flow/cold_reset/fmc_alias.rs [hunk 1]
```diff
@@ -252,6 +252,6 @@ impl FmcAliasLayer {
             flags |= dice::FLAG_BIT_DEBUG;
         }
 
-        flags.to_be_bytes()
+        flags.reverse_bits().to_be_bytes()
```
Comment (jhand2) [inline rom/dev/src/flow/cold_reset/fmc_alias.rs:255]: Why do the bits need to be reversed? The constants seem to match the spec.

File: test/tests/caliptra_integration_tests/smoke_testdata/rt_alias_cert_redacted.txt [hunk 2]
```diff
@@ -30,7 +30,7 @@ Certificate:
             2.23.133.5.4.4: 
                 0....................
             2.23.133.5.4.1: 
-                DDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD
+                DDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD
```
Comment (jhand2) [inline test/tests/caliptra_integration_tests/smoke_testdata/rt_alias_cert_redacted.txt:33]: What's the change that leads to the TCBInfo extension being shorter?

File: rom/dev/src/flow/cold_reset/fmc_alias.rs (hunk 1 above)
Comment (nquarton) [reply rom/dev/src/flow/cold_reset/fmc_alias.rs:255]: Our understanding for a DER encoded ASN1 bit string is that for the actual value (the flags) the most-significant-bit of the first byte transferred corresponds to bit 0 of the flags.

See this example: https://learn.microsoft.com/en-us/windows/win32/seccertenroll/about-bit-string

File: test/tests/caliptra_integration_tests/smoke_testdata/rt_alias_cert_redacted.txt (hunk 2 above)
Comment (nquarton) [reply test/tests/caliptra_integration_tests/smoke_testdata/rt_alias_cert_redacted.txt:33]: This is the removal of Vendor="Caliptra" and Model="RT" from this info.

**Review by fdamato — APPROVED** [review]

File: rom/dev/src/flow/cold_reset/fmc_alias.rs (hunk 1 above)
Comment (jhand2) [reply rom/dev/src/flow/cold_reset/fmc_alias.rs:255]: Ah, you're right. Copying from our chat discussion

Looking at the ASN.1 spec: https://www.itu.int/ITU-T/studygroups/com17/languages/X.680-0207.pdf

Section 21.2
> The first bit in a bit string is called the leading bit. The final bit in a bit string is called the trailing bit.

Section 21.4
> The leading bit of the bit string is identified by the "number" zero, with succeeding bits having successive values.

**Review by jhand2 — APPROVED** [review]

File: rom/dev/src/flow/cold_reset/fmc_alias.rs (hunk 1 above)
Comment (nquarton) [reply rom/dev/src/flow/cold_reset/fmc_alias.rs:255]: Oh nice this is the point of truth we needed. I was only finding references that implied this. Thanks for getting to the bottom of this!

**Review by mhatrevi — APPROVED** [review]

**Review by JohnTraverAmd — APPROVED** [review]

**Pull request merged**